Makefile

```diff
@@ -51,8 +54,10 @@ test-simplecov:
 	docker build -f examples/simplecov/Dockerfile .
 
 publish:
-	$(AWS) s3 sync --acl public-read artifacts/bin s3://codeclimate/test-reporter
+	$(AWS) s3 cp --acl public-read --recursive artifacts/bin/ s3://codeclimate/test-reporter/ --exclude "*" --include "test-reporter-$(VERSION)*" --include "test-reporter-latest*"
```

Comment:
Possibly. How about you add that snippet to #88 as an option? I'd like to think about it a bit more before actioning anything.